src/components/FilmsContainer/FilmsContainer.js

```diff
@@ -2,24 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import FilmCard from '../FilmCardComponent/';
 
-const mockFilms = [
-    {
-        poster_path: '123123.jpg',
-        genres: [15, 28],
-        release_date: '2020-12-12',
-        title: 'TEST',
-    },
-];
-
-function FilmsContainer() {
+function FilmsContainer(props) {
     return (
```

Comment:
you can remove return statements and simplify your function component

Reply:
Fixed